lib/Account.php

```diff
@@ -209,21 +197,6 @@ protected function listMailboxes($pattern = '*') {
 		return $mailboxes;
 	}
 
-	/**
-	 * @param string $folderId
-	 * @return Mailbox
-	 */
-	public function getMailbox($folderId) {
```

Comment:
Not yet. I think this method is used in many places. Please only remove `createMailbox`

Reply:
Thanks @ChristophWurst ! Updated in https://github.com/nextcloud/mail/pull/2056/commits/14e2a49057ff613e227adca0aef3a9f2a14ec94b